### DISCUSSION TITLE
Just some feedback (EDITED)

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#3631] [Codespaces] [answered]
I like this. It's a nice place for maintainers to communicate with users. Just once suggestion, add a better Markdown editor. You know, something like VSCode's with syntax highlighting. It may be difficult to implement but it's worth a try! Other than that, I think I will be enabling this on each of my repositories.

EDIT: So to "elaborate my issue/suggestion more", I mean like the editor for files on GitHub. So like when you have a bigger space. Plus maybe a preview on the side to see what it looks like live instead of having a separate tab.

**Comment by PranamBhat** [comment]
It would be great if you can elaborate your issue/suggestion more!

**Reply by TechStudent10** [reply]
@PranamBhat I edited it.

**Comment by PranamBhat** [comment, marked as answer]
Hi @TechStudent11 

There are certain criteria's GitHub is following.

For example, you can take look at preview over comments,etc.

But when it comes to a file, GitHub is meant for write a code on your editor (It is up to you - whichever editor you need for you project. for example, VS Code for Angular and for Java you prefer Eclipse and so on) and push to GitHub.

GitHub is not meant for writing/editing code on your repo. It will be like a blog if so!!!

Hope you understand! Thank you.

**Reply by TechStudent10** [reply]
@PranamBhat oh well. Thanks for replying.
